build-image/first-boot.bash

```diff
@@ -149,29 +151,29 @@ if hash python3 2>/dev/null; then bash /boot/webif.bash reinsure_running; fi
 echo -n "$(timestamp) [openHABian] Installing git package... "
 if apt-get -y install git &>/dev/null; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
 
-echo -n "$(timestamp) [openHABian] Cloning myself from ${REPOSITORYURL}, ${CLONEBRANCH} branch... "
 if [ -d /opt/openhabian/ ]; then cd /opt && rm -rf /opt/openhabian/; fi
-if git clone -q -b "$CLONEBRANCH" "$REPOSITORYURL" /opt/openhabian; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+# shellcheck disable=SC2154
+echo -n "$(timestamp) [openHABian] Cloning myself from ${repositoryurl}, ${clonebranch} branch... "
+if git clone -q -b "$clonebranch" "$repositoryurl" /opt/openhabian; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
 ln -sfn /opt/openhabian/openhabian-setup.sh /usr/local/bin/openhabian-config
 
-echo "$(timestamp) [openHABian] Executing 'openhabian-setup.sh unattended'... "
-if (/bin/bash /opt/openhabian/openhabian-setup.sh unattended); then
-#if (/bin/bash /opt/openhabian/openhabian-setup.sh unattended_debug); then
-  systemctl start openhab2.service
+# shellcheck disable=SC2154
+echo "$(timestamp) [openHABian] Executing openhabian-setup.sh ${mode}... "
+if (/bin/bash "$BASEDIR"/openhabian-setup.sh "$mode"); then
```

Comment:
when we drop init.bash -> back to hard coded (anyway hard coded below)

Reply:
ok